content/software-security/compliance/cmmc-2/cmmc-chainguard.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Simplify Your Path to CMMC 2.0 Compliance with Chainguard"
+description: "Simplify Your Path to CMMC 2.0 Compliance with Chainguard"
+lead: "Chainguard Images reduce the time and effort for establishing CMMC 2.0 compliance."
+type: "article"
+date: 2024-08-09T19:10:09+00:00
+lastmod: 2024-08-09T19:10:09+00:00
+contributors: ["Erin Glass"]
+draft: false
+tags: ["compliance", "CMMC 2.0", "standards"]
+images: []
+menu:
+  docs:
+    parent: "cmmc-2"
+weight: 005
+toc: true
+---
+
+Achieving Cybersecurity Maturity Model Certification (CMMC) 2.0 Level 2 or Level 3 certification can be a complex and resource-intensive process, particularly for organizations managing containerized environments and addressing vulnerabilities. Chainguard simplifies this journey by offering specialized solutions that drastically reduce the time and effort needed to meet compliance requirements. Our FIPS-compliant (Federal Information Processing Standard) images, combined with detailed SBOM (Software Bill of Materials) and STIG-hardened (Security Technical Implementation Guide) configurations, provide a strong foundation for meeting the requirements of CMMC 2.0.
+
+## What are STIG-Hardened FIPS Images?
+
+STIG-hardened FIPS images are pre-configured container images that have been secured according to the Security Technical Implementation Guide (STIG) standards set by the Defense Information Systems Agency (DISA). These images are designed to meet stringent federal security requirements, combining FIPS-compliant encryption with robust security configurations that protect against vulnerabilities and threats. By using STIG-hardened FIPS images, organizations ensure that their systems adhere to federal encryption standards and best practices for cybersecurity, making them particularly valuable in environments that require high levels of security, such as those governed by CMMC 2.0.
```

Comment:
Why "...are designed to..." instead of just "meet"? Are we worried they do not? Or are we hedging our bets to try to avoid accidental legal contracts?